genesis/devnet/genesis.json

```diff
@@ -291,8 +291,8 @@
     "distribution": {
       "params": {
         "community_tax": "0.000000000000000000",
-        "base_proposer_reward": "0.010000000000000000",
-        "bonus_proposer_reward": "0.040000000000000000",
+        "base_proposer_reward": "0.000000000000000000",
+        "bonus_proposer_reward": "0.000000000000000000",
```

Comment:
this was override be generate.sh, what is the correct value here?

Reply:
looks ok, but I am not sure

Reply:
pls ask @mikluke

Reply:
It's okay, I've changed defaults in code